commit d6c6187b5d522d3b079cefa0ad2320e428652a4e
Author: Sven Neumann <s.neumann@raumfeld.com>
Date:   Tue May 25 10:30:31 2010 +0200

    Mark gupnp_didl_lite_object_new_from_xml() as internal
    
    Signed-off-by: Sven Neumann <s.neumann@raumfeld.com>

diff --git a/libgupnp-av/gupnp-didl-lite-object-private.h b/libgupnp-av/gupnp-didl-lite-object-private.h
index a776cd5..2016ea4 100644
--- a/libgupnp-av/gupnp-didl-lite-object-private.h
+++ b/libgupnp-av/gupnp-didl-lite-object-private.h
@@ -28,7 +28,7 @@
 
 G_BEGIN_DECLS
 
-GUPnPDIDLLiteObject *
+G_GNUC_INTERNAL GUPnPDIDLLiteObject *
 gupnp_didl_lite_object_new_from_xml     (xmlNode     *xml_node,
                                          GUPnPXMLDoc *xml_doc);
 
